server/services/snapshot/headstart_snapshot.php

```diff
@@ -20,24 +20,24 @@
             if (<?php echo json_encode($_GET['service']) ?> === "linkedcat" ||
                 <?php echo json_encode($_GET['service']) ?> === "linkedcat_authorview" ||
                 <?php echo json_encode($_GET['service']) ?> === "linkedcat_browseview") {
-                if(<?php echo json_encode($_GET['vis_mode']) ?> === "authors") {
+                if(<?php echo json_encode(isset($_GET['vis_mode']) && $_GET['vis_mode']) ?> === "authors") {
```

Comment:
This should rather be something like
`if (isset($_GET['vis_mode'])) echo $_GET['vis_mode']`

Otherwise, it will output `true` or `false` rather than `authors` or `browse`

Reply:
It may be worth doing this once at the beginning of the file and then only echo the variable the parameter was assigned to at this point.